pkgs/misc/cups/drivers/fxlinuxprintutil/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, lib, fetchzip, dpkg, autoPatchelfHook, cups, tcl, tk, xorg, makeWrapper }:
+let
+  debPlatform =
+    if stdenv.hostPlatform.system == "x86_64-linux" then "amd64"
+    else if stdenv.hostPlatform.system == "i686-linux" then "i386"
+         else throw "Unsupported system: ${stdenv.hostPlatform.system}";
+in
+stdenv.mkDerivation rec {
+  name = "fxlinuxprintutil-${version}";
```

Comment:
```suggestion
  pname = "fxlinuxprintutil";
```

Reply:
when `pname` and `version` attributes are present they get concatenated like `${pname}-${version}` to create `name` automagically.

Reply:
done in d03c4109d699670f1bfe11794378c82ee7d26499